### PR TITLE
Purpur > Paper

### PR DESCRIPTION
**Pull request by zax71**

### CODE REVIEW

**Review by Evidentsinger14 — CHANGES_REQUESTED** [review]
> For the Bukkit platform you mave many 
```diff
- mave
+ have
```
> If you want peformance tewaks on top that can be powerful but may break the more technical mechanics or quality of life tweaks try 
```diff
- If you want peformance tewaks on top that can be powerful but may break the more technical mechanics or quality of life tweaks try [Purpur](https://purpurmc.org/downloads).
+ If you want performance tweaks that can be powerful, but may break the more technical mechanics or quality of life tweaks try [Purpur](https://purpurmc.org/downloads).
```

**Comment by Evidentsinger14** [comment]
@zax71 i didn't see any other spelling/grammar errors, but i'll have another look in a few minutes.

File: docs/running_a_server/optimization.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ re just going to outline the two best known "paths", the bukkit platform, and fa
 The gist of it is the bukkit platform is generally recommended for bigger public servers and fabric for smaller servers with friends.  
 The bukkit platform allows you to use plugins, and fabric allows you to run mods.
 
-For the bukkit platform the [Paper](https://papermc.io/) jar is recommended, it includes many performance improvements, bug fixes and configuration options  
+For the Bukkit platform you mave many options for a server jar, the most stable of which is [Paper](https://papermc.io). Paper includes many performance improvements, bug fixes and configuration options. If you want peformance tewaks on top that can be powerful but may break the more technical mechanics or quality of life tweaks try [Purpur](https://purpurmc.org/downloads).
```
Comment (democracycraft) [inline docs/running_a_server/optimization.md:47]: Small typo here, I think you meant 'have' not 'mave'!

**Comment by zax71** [comment]
Thanks for pointing that out, I’ll fix it when I get time!

**Pull request merged**